api/src/main/java/org/openmrs/module/pihcore/setup/LocationTagSetup.java

```diff
@@ -74,23 +74,18 @@ private static void setupLocationTagsForSierraLeone(LocationService locationServ
 
     private static void setupLocationTagsForMexico(LocationService locationService, Config config) throws NoSuchFieldException, IllegalAccessException  {
 
-        Field locationField = MexicoLocations.class.getField(config.getSite().name().toString());
+        // Get the current location -- each site only has one available
+        Field locationField = MexicoLocations.class.getField(config.getSite().name());
         LocationDescriptor location = (LocationDescriptor) locationField.get(LocationDescriptor.class);
 
-        // Tags for our hub in Jaltenango
-        List<LocationDescriptor> jalte = Arrays.asList(MexicoLocations.JALTENANGO);
-        setLocationTagsFor(locationService, LocationTags.LOGIN_LOCATION, jalte);
-        setLocationTagsFor(locationService, LocationTags.MEDICAL_RECORD_LOCATION, jalte);
```

Comment:
I assume that removing these and the NCD consult location was intentional?

Reply:
Yes—there is no Jaltenango location. Actually, I should remove that from MexicoLocations as well.